cpp/unit/unit.cpp

```diff
@@ -106,16 +107,20 @@ void Unit::update_secondary(int64_t time_elapsed) {
 }
 
 void Unit::draw() {
+	this->draw(this->location, this->graphics);
```

Comment:
Ill add the check then, I thought it was there... It wouldnt have any affect atm as draw calls originate from terrain (to get the draw order) so a drawn unit always has a location